packages/schematics/angular/migrations/update-12/update-web-workers.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { DirEntry, Rule, UpdateRecorder } from '@angular-devkit/schematics';
+import * as ts from '../../third_party/github.com/Microsoft/TypeScript/lib/typescript';
+
+function* visit(directory: DirEntry): IterableIterator<ts.SourceFile> {
+  for (const path of directory.subfiles) {
+    if (path.endsWith('.ts') && !path.endsWith('.d.ts')) {
+      const entry = directory.file(path);
+      if (entry) {
+        const content = entry.content;
+        if (content.includes('Worker')) {
```

Comment:
should we check `new Worker(` instead to reduce the likelihood of processing unnecessary files?

Reply:
There can be an arbitrary amount of whitespace between `new` and `Worker` which would require a regular expression and negatively affect performance.  `includes` is significantly faster than using a regular expression.